python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/dot1x.j2

```diff
@@ -36,15 +36,46 @@
 {%             set hold_period = dot1x.mac_based_authentication.hold_period | arista.avd.default('-') %}
 | {{ delay }} | {{ hold_period }} |
 {%         endif %}
+{%         if dot1x.radius_av_pair_username_format is arista.avd.defined %}
+
+#### 802.1X Radius AV pair username format
+
+| Delimiter | MAC String Case |
+| --------- | --------------- |
+| {{ dot1x.radius_av_pair_username_format.delimiter | arista.avd.default("-") }} | {{ dot1x.radius_av_pair_username_format.mac_string_case | arista.avd.default("-") }} |
+{%         endif %}
 {%         if dot1x.radius_av_pair is arista.avd.defined %}
 
 #### 802.1X Radius AV pair
 
-| Service type | Framed MTU |
-| ------------ | ---------- |
+| Service type | Framed MTU | LLDP System Name | LLDP System Request List | DHCP Hostname | DHCP Parameter Request List | DHCP Vendor Class ID | Filter Delimiter Period | Filter IPv4 IPv6 | Filter Multiple |
+| ------------ | ---------- | ---------------- | ------------------------ | ------------- | --------------------------- | -------------------- | ----------------------- | ---------------- | --------------- |
```

Comment:
Since all the attributes are independent, we can have different tables for LLDP, DHCP and Filter for better readability.